src/components/Login/Login.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import  {useDispatch}  from "react-redux";
+import { loginAction } from "../../store/reducers/userReducer";
+import { Avatar, Button, Grid, Link, TextField, Typography, Box, Container } from "@mui/material";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+
+const Login: React.FunctionComponent = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <Box
+        sx={{
+          marginTop: 8,
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+        }}
+      >
+        <Avatar sx={{ m: 1, bgcolor: "primary.dark" }}>
+          <LockOutlinedIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
+          Log in
+        </Typography>
+        <Box component="form" sx={{ mt: 1 }}>
+          <TextField
+            margin="normal"
+            required
+            fullWidth
+            id="email"
+            label="Email Address"
+            name="email"
+            autoFocus
+          />
+          <TextField
+            margin="normal"
+            required
+            fullWidth
+            name="password"
+            label="Password"
+            type="password"
+            id="password"
+          />
+
+          <Button
+            onClick={() => dispatch(loginAction())}
+            type="submit"
+            fullWidth
+            variant="contained"
+            sx={{ mt: 3, mb: 2 }}
+          >
+            Log In
+          </Button>
+          <Grid container>
+            <Grid item xs>
+              <Link href="#" variant="body2">
+                Forgot password?
+              </Link>
+            </Grid>
+            <Grid item>
+              <Link href="#" variant="body2">
+                {"Don't have an account? Sign Up"}
```

Comment:
in my opinion, curly braces and quotes are not needed here, and it will work correctly.

Reply:
done. write Dont instead of Don't

Reply:
why? Leave don't or do not

Reply:
fixed